test/Google.Api.Gax.Tests/RetryTest.cs

```diff
@@ -31,8 +32,30 @@ public class RetryTest
             MaxDelay = TimeSpan.FromTicks(6000) // Irrelevant
         };
 
-        [Fact]
-        public void FirstCallSucceeds()
+        private static Task<TResponse> Call<TRequest, TResponse>(
```

Comment:
In my very humble opinion you're being perhaps maybe just a tiny bit too picky ;)
But please feel free to correct me on this -  and I'll change to code faster than a speeding async call.